cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/KafkaSampleStore.java

```diff
@@ -384,6 +390,7 @@ public void run() {
             }
           }
         }
+        LOG.info("Metric loader finished loading samples.");
```

Comment:
There are multiple metric loaders (i.e. as many as the number of consumers) . Can we add an identifier here to say something like `5 out of 8 metric loaders finished loading samples.`?

Reply:
When this message is printed, the LOG4J does have the information of which  metric loader  thread(format like pool-`total_thread_number`-thread-`current_thread_number`) prepending to this message, so we can distinguish the threads and a `grep` command can help us locate which thread is actual lagging. I feel to print message like `5 out of 8` we need to pass more args to Runnable and needs a atomic variable sync between worker thread, which is kinda too much for just logging/debugging purpose.